Functions.Templates/ProjectTemplate/CSharp-Isolated/Program.cs

```diff
@@ -0,0 +1,26 @@
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Azure.Functions.Worker.Configuration;
+
+namespace Company.FunctionApp
+{
+    class Program
```

Comment:
Use this file as an example. Replace program with SourceName value. Match the sourceName in respective locations. https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Templates/CosmosDBTrigger-CSharp/CosmosDBTriggerCSharp.cs

Reply:
This would still be `Program`. This is the main file (new to this model).